projects/components/src/toggle-group/toggle-group.component.test.ts

```diff
@@ -0,0 +1,49 @@
+import { fakeAsync } from '@angular/core/testing';
+import { LabelComponent, ToggleGroupComponent, ToggleItem } from '@hypertrace/components';
+import { byText, createHostFactory, Spectator } from '@ngneat/spectator/jest';
+import { ToggleItemComponent } from './toggle-item.component';
+
+describe('Toggle Group Component', () => {
+  let spectator: Spectator<ToggleGroupComponent>;
+
+  const createHost = createHostFactory({
+    component: ToggleGroupComponent,
+    shallow: true,
+    declarations: [ToggleItemComponent, LabelComponent]
```

Comment:
Why does this need the real label component? I'm guessing it's because we're verifying the text through how the label component renders it. Instead, in a shallow test, bests practice would be to mock it, and verify the label's input is set to whaat you expect. This means that if label were to change (say, add a new component internally) this test won't break. See the paginator test if you'd like an example - that mocks label.